packages/ui/src/codebox/index.tsx

```diff
@@ -1,13 +1,18 @@
 import React, { useState } from 'react';
-import { Highlighter } from '../highlighter';
+import { Highlighter, HighlighterProps } from '../highlighter';
 import { Box, BoxProps } from '../box';
 
 export const CodeBlock = ({
   code,
   showLineNumbers,
   style = {},
+  language,
   ...rest
-}: { code: string; showLineNumbers?: boolean } & BoxProps) => {
+}: {
```

Comment:
Would be great if this could be created as an interface `CodeBlockProps` above

Reply:
sure thing!